packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -195,17 +198,20 @@ export class MirrorNodeClient {
         let ms;
         try {
             let response;
-            const headers = {
+
+            controller = new AbortController();
```

Comment:
Q: Checking my js node knowledge here but will concurrent runs ever result in 1 request that was aborted affecting a new request ?
If so would having the `controller1 variable be scoped to the function and not the class be safer?

Reply:
Done